extension/source/QuillPage/components/WorkflowNumbers.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+import Range from '../../helpers/Range';
+
+const WorkflowNumbers: React.FunctionComponent<{
+  current: number;
+  max: number;
+  onSelect?: (selection: number) => void;
+}> = ({ current, max, onSelect = () => {} }) => (
+  <div className="workflow-numbers quick-row">
+    {Range(max).map((i) => (
+      <>
```

Comment:
```suggestion
      <div key={i}>
```

needs a key

Reply:
Ohh nice. I wasn't sure how to resolve that one that react was complaining about.

Done 👍.